algorithms.py

```diff
@@ -1148,6 +1150,22 @@ def print_heaps(self, L):
                 u = u.next
             print(s)
 
+    def census_event(self, time):
+        for pop in self.P:
+            # pop.print_state()
+            for a in pop._ancestors:
+                for s in a:
+                    while s is not None:
+                        self.flush_edges()
+                        u = self.tables.nodes.add_row(time=time, flags=18, 
```

Comment:
This will be something like ``flags=msprime.NODE_IS_CENSUS_EVENT``. But, it's the bit position 18 we're setting anyway, so for now it's ``1<<18``.